package.json

```diff
@@ -55,8 +55,8 @@
     "react-virtualized": "9.20.1"
   },
   "peerDependencies": {
-    "react": ">=15.4.1 <16.9.0 || 16.7.0-alpha.0",
-    "react-dom": ">=15.4.1 <16.9.0 || 16.7.0-alpha.0"
+    "react": ">=16.2.0 <16.9.0 || 16.7.0-alpha.0",
```

Comment:
that's technically a breaking change 👼 

https://www.conventionalcommits.org/en/v1.0.0-beta.2/#commit-message-with-description-and-breaking-change-in-body

Reply:
Ah, you're right! I'll update the commit with the appropriate information.

We're using React 16+ in the Mesosphere projects that use ui-kit, so it shouldn't be a problem to upgrade